services/registry/Migrations/DbInitializer.cs

```diff
@@ -13,132 +14,19 @@ public static void Initialize(ShareContext context)
-                new ShareModel{TickerSymbol = "MAERSK A", Owner = 5},
-                new ShareModel{TickerSymbol = "MAERSK A", Owner = 5},
-                new ShareModel{TickerSymbol = "MAERSK A", Owner = 5},
-                new ShareModel{TickerSymbol = "MAERSK A", Owner = 5},
-                new ShareModel{TickerSymbol = "MAERSK A", Owner = 5},
-                new ShareModel{TickerSymbol = "MAERSK A", Owner = 5},
-                new ShareModel{TickerSymbol = "MAERSK A", Owner = 5},
-                new ShareModel{TickerSymbol = "MAERSK A", Owner = 5},
-                new ShareModel{TickerSymbol = "MAERSK A", Owner = 5},
-                new ShareModel{TickerSymbol = "MAERSK A", Owner = 5},
-                new ShareModel{TickerSymbol = "MAERSK A", Owner = 5},
-                new ShareModel{TickerSymbol = "MAERSK A", Owner = 5},
-                new ShareModel{TickerSymbol = "MAERSK A", Owner = 5},
-                new ShareModel{TickerSymbol = "DENERG", Owner = 6},
-                new ShareModel{TickerSymbol = "DENERG", Owner = 6},
-                new ShareModel{TickerSymbol = "DENERG", Owner = 6},
-                new ShareModel{TickerSymbol = "DENERG", Owner = 6},
-                new ShareModel{TickerSymbol = "DENERG", Owner = 6},
-                new ShareModel{TickerSymbol = "DENERG", Owner = 6},
-                new ShareModel{TickerSymbol = "DENERG", Owner = 6},
-                new ShareModel{TickerSymbol = "DENERG", Owner = 6},
-                new ShareModel{TickerSymbol = "DENERG", Owner = 6},
-                new ShareModel{TickerSymbol = "DENERG", Owner = 6},
-                new ShareModel{TickerSymbol = "DENERG", Owner = 6},
-                new ShareModel{TickerSymbol = "DENERG", Owner = 6},
-                new ShareModel{TickerSymbol = "DENERG", Owner = 6},
-                new ShareModel{TickerSymbol = "DENERG", Owner = 6},
-                new ShareModel{TickerSymbol = "DENERG", Owner = 6},
-                new ShareModel{TickerSymbol = "DENERG", Owner = 6},
-                new ShareModel{TickerSymbol = "DENERG", Owner = 6},
-                new ShareModel{TickerSymbol = "DENERG", Owner = 6},
-                new ShareModel{TickerSymbol = "DENERG", Owner = 6},
-                new ShareModel{TickerSymbol = "DENERG", Owner = 6},
-            };
+                sharesList.Add(new ShareModel { TickerSymbol = "NVO", Owner = 1 });
+                sharesList.Add(new ShareModel { TickerSymbol = "GOOG", Owner = 2 });
+                sharesList.Add(new ShareModel { TickerSymbol = "VWS", Owner = 3 });
+                sharesList.Add(new ShareModel { TickerSymbol = "DANSKE", Owner = 4 });
+                sharesList.Add(new ShareModel { TickerSymbol = "MAERSK A", Owner = 5 });
+                sharesList.Add(new ShareModel { TickerSymbol = "DENERG", Owner = 6 });
+            }
 
-            foreach (var share in shares)
+            foreach (var share in sharesList)
```

Comment:
Another review improvement. Additional 11% has been added to quality of previously created tasks.